File: two_sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    var ans=[]
    for (var i =0;i<nums.length;i++ ){
        for (var j = 0;j<nums.length-1;j++ ){
            if( i!=j ){
                var sum = nums[i]+nums[j];
                if (sum==target){
                    ans.push(i)
                    ans.push(j)
                    i=nums.length+1
                    break;
                }
            }
        }
    }
    return ans
};

File: twoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        nums_dict={}
        if len(nums)==2:
            return [0,1]
        for i in range(len(nums)):
            if nums_dict.get(target-nums[i])!=None:
                return [nums_dict.get(target-nums[i]),i];
            else:
                nums_dict[nums[i]]=i;
